### ISSUE TITLE
Reload settings on change

### ISSUE THREAD

**Issue by JackGruber** [JackGruber/joplin-plugin-notelistpreview#3]
Reload settings if settings have been changed

**Comment by JackGruber** [comment]
Update or registration of the Renderer is not currently offered by the API

**Comment by ivangretsky** [comment]
This would be awesome. Is there a proposal for such an API? Can I thumb it up?

**Comment by JackGruber** [comment]
No, currently there is no proposal or tracker that this will be implemented on Joplin side
